### ISSUE TITLE
[CRASH] Fast Track crashes when trying to load a save.

### ISSUE THREAD

**Issue by LeonusDH** [peterhaneve/ONIMods#408]
## Expected Behavior
Loading a save..?
I'm playing with the DLC.

## Current Behavior
When creating a new game, everything was fine, but when I tried to load a save, I got a crash.

## Mod Versions
Fast Track Beta - 0.12.2.0

## Steps to Reproduce
1. Run the game
2. Load last save
3. Crash

## Output Log
In game crash: https://pastebin.com/0cg137jp
Player.log: https://gist.github.com/LeonusDH/324616a4282617cdda7ec295d023cd51
(I used gist as the file turned out to be very large.)

**Comment by affo85** [comment]
TLDR: Neither 0.12.1.0 or 0.12.2.0 works with Public Beta u49 after a small steam update today (2023-10-04).

Same here. Been working great until today. Game crashes on startup (before main menu).

I've been playing the Public test beta u49 for the last couple of weeks with Fast Track 0.12.1.0, but today there was a minor update on Steam. (Also, the public test beta u49 have had 1-2 more updates the last couple of weeks as well).

Have tried this new 0.12.2.0 as well, but it also crashes the game on startup. And both versions become marked in red font in the mod manager on the main menu after a reboot with them disabled.

Any tips much appreciated. My fps tanks without this lovely mod :)

**Comment by peterhaneve** [comment]
Public Testing branches are not supported by Fast Track. If Fast Track is important to you, consider staying on the stable branch for now.

**Comment by LeonusDH** [comment]
> Public Testing branches are not supported by Fast Track. If Fast Track is important to you, consider staying on the stable branch for now.

I got my error on the stable branch!
I don't play on testing branches.

**Comment by LeonusDH** [comment]
> TLDR: Neither 0.12.1.0 or 0.12.2.0 works with Public Beta u49 after a small steam update today (2023-10-04).
> 
> Same here. Been working great until today. Game crashes on startup (before main menu).
> 
> I've been playing the Public test beta u49 for the last couple of weeks with Fast Track 0.12.1.0, but today there was a minor update on Steam. (Also, the public test beta u49 have had 1-2 more updates the last couple of weeks as well).
> 
> Have tried this new 0.12.2.0 as well, but it also crashes the game on startup. And both versions become marked in red font in the mod manager on the main menu after a reboot with them disabled.
> 
> Any tips much appreciated. My fps tanks without this lovely mod :)
> 
> Edit: I understand that Public beta branches aren't supported by this mod. FWIW, I will append my player log anyway.
> 
> [Player.log](https://github.com/peterhaneve/ONIMods/files/12809204/Player.log)
> 
> Best Regards /Rasmus

My issue has nothing to do with your error....

**Comment by affo85** [comment]
> 
> My issue has nothing to do with your error....

I understand. I just wanted to share if Klei pushed a similar update today to the official build version.

**Comment by affo85** [comment]
> Public Testing branches are not supported by Fast Track. If Fast Track is important to you, consider staying on the stable branch for now.

Thank you! I will do that from now on.
I have learned my lesson this time ^^

I was just hoping if anyone had any advice, because now I'm stuck without this great mod on my first 3000 cycle save :)

**Comment by peterhaneve** [comment]
> > Public Testing branches are not supported by Fast Track. If Fast Track is important to you, consider staying on the stable branch for now.
> 
> I got my error on the stable branch! I don't play on testing branches.

For this error, it appears to be a mod conflict or crash with Ony's Research Manager:
```
System.NullReferenceException: Object reference not set to an instance of an object
  at Ony.OxygenNotIncluded.ResearchManager.ResearchTrackerComponent.UpdateActive () [0x0019b] in <9b5fb67ef9d149d784f1411d3095d4ea>:0 
  at Ony.OxygenNotIncluded.ResearchManager.ResearchTrackerComponent.Activate () [0x0000a] in <9b5fb67ef9d149d784f1411d3095d4ea>:0 
  at Ony.OxygenNotIncluded.ResearchManager.ResearchScreen_OnSpawn.Postfix (ResearchScreen __instance, System.Boolean show) [0x0000c] in <9b5fb67ef9d149d784f1411d3095d4ea>:0 
```
